aviatrix/utils.go

```diff
@@ -298,8 +299,16 @@ func checkPAVMVersionFormat(version string) string {
 
 // compareVersion compares two Semantic Versions
 func compareVersion(version1, version2 string) bool {
-	v1, _ := version.NewVersion(version1)
-	v2, _ := version.NewVersion(version2)
+	v1, err := version.NewVersion(version1)
+	if err != nil {
+		log.Println("not support sort this version format: ", version1, err)
```

Comment:
This error message should be reworded and should probably use `log.Printf`. Something like `log.Printf(unsupported firewall image version format: %s, version1)`

Reply:
thank you, have fixed